appdev/guides/dependencies.rst

```diff
@@ -92,45 +88,66 @@ If the library is not in the Ubuntu Repositories, but in a PPA, you can add the
       ]
     }
 
-Otherwise you may need to add a script to download the pre-built library.
+Otherwise add a script to download the pre-built library.
 
 Usage
 -----
-First, you may need to specify the include directory where the compiler can find the headers. Second, you need to link the library itself against your app's binary, except it is a header library, where all the source code is located in header files.
+First, you need to specify the include directory where the headers can be found
+that you include into the app's source code. Second, you need to link the
+library's binary against your app's binary.
```

Comment:
There is no need to perform an arbitrary line break in this repository. Any tool telling you to break at 100 characters is lying.